src/graph/GraphViewServer.ts

```diff
@@ -436,6 +436,7 @@ export class GraphViewServer extends EventEmitter {
   private handleQueryMessage(queryId: number, gremlin: string) {
     this.log(`Query requested: queryId=${queryId}, gremlin="${gremlin}"`);
 
+    //tslint:disable-next-line:no-floating-promises
     this.queryAndShowResults(queryId, gremlin);
```

Comment:
`handleQueryMessage` is not async, and is used in the GraphViewServerSocket. 
@StephenWeatherford should this be awaited?

Reply:
Could you add return value to handleQueryMessage?

Makes sense, can't handle this synchronously.

FYI, that would normally be a red flag - it means we could get overlapping calls.  Luckily this is already being handled at least somewhat - I associate each query with an ID, and the client ignores results that are not from the most recent query, so we're probably fine here.